
class EndOfExamException(Exception):
    pass


class ReapetedTask(Exception):
    pass
